lib/client.js

```diff
@@ -309,6 +317,41 @@ extend(Raven.prototype, {
     utils.consoleAlert('setTagsContext has been deprecated and will be removed in v2.0');
     this._globalContext.tags = extend({}, this._globalContext.tags, tags);
     return this;
+  },
+
+  setCallbackHelper: function (propertyName, callback) {
+    var original = this[propertyName];
+    if (typeof callback === 'function') {
+      this[propertyName] = function (data) {
+        return callback(data, original);
```

Comment:
Can you add a test that verifies that `original` is passed if there is a pre-existing callback?